pkg/util/util.go

```diff
@@ -185,3 +185,13 @@ func EnsureDirExists(dir string) error {
 
 	return err
 }
+
+func ValidBackend(validBackends []string, backend string) bool {
```

Comment:
does this really need to exist here? Isn't this a generic find in slice?

Reply:
I will resolve the comment but please re-open if you don't like the implementation